db/migrations/004.do.active-deals-submitted-at.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE active_deals ADD COLUMN IF NOT EXISTS submitted_at TIMESTAMP;
+
+CREATE INDEX IF NOT EXISTS active_deals_activated_at_epoch ON active_deals (activated_at_epoch);
```

Comment:
I am wondering if I should move this to separate migration file?

Reply:
We usually create a single migration script per pull request, unless there is a good reason for splitting.

I vaguely remember that postgrator runs each script in one transaction, and certain alterations cannot be executed together in one transactions, so we had to move them to a new migration script. If that was the case, you would know, because `npm run migrate` would fail.